src/Tizen.Wearable.CircularUI.Forms.Renderer/TwoButtonPageRenderer.cs

```diff
@@ -115,12 +116,12 @@ void UpdateSecondButton(bool initialize)
                 if (Element.SecondButton is ColorMenuItem)
                 {
                     var item = ((ColorMenuItem)Element.SecondButton);
-                    _widget.ShowButton2(item.Text, item.Icon, item.BackgroundColor.ToNative(), () => ((IMenuItemController)item).Activate());
+                    _widget.ShowButton2(item.Text, item.IconImageSource, item.BackgroundColor.ToNative(), () => ((IMenuItemController)item).Activate());
```

Comment:
To minimize the change, you can just change below line without other changes.
```suggestion
_widget.ShowButton2(item.Text, (item.IconImageSource as FileImageSource), item.BackgroundColor.ToNative(), () => ((IMenuItemController)item).Activate());
```